plugin/storage/cassandra/spanstore/writer.go

```diff
@@ -181,7 +186,10 @@ func (s *SpanWriter) writeIndexes(span *model.Span, ds *dbmodel.Span) error {
 		// should this be a soft failure?
 		return s.logError(ds, err, "Failed to insert service name and operation name", s.logger)
 	}
+	return nil
+}
 
+func (s *SpanWriter) writeOtherIndexes(span *model.Span, ds *dbmodel.Span) error {
```

Comment:
Could you use a more descriptive name here?

Reply:
like what?

Reply:
Perhaps this could just be `writeIndexes` like earlier, but it calls `writeServiceOperationIndex`? Might simplify the branching in `writer.go` as well.